### PR TITLE
Parallel computing notes to wiki. Upload misc project.

### PR DESCRIPTION
**Pull request by dougwoods3**
The parallel computing notes need some clean up (and more discussion).

I want to see how the movie links to the site.

Other stuff is just updating my page.

### CODE REVIEW

File: wiki/rogue/performance.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Some performance marks about ne.rogue:
```
Comment (albeanth) [inline wiki/rogue/performance.md:1]: You could use the printinfo.cpp file to display the results of the performance specs of rogue. And then you could break down what the table outputs. Sort of link the two together.

File: wiki/rogue/compilers.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+The compilers available on ne.rogue are:
+* GCC
+* Clang
+* ICC?
```
Comment (albeanth) [inline wiki/rogue/compilers.md:4]: I would put a disclaimer than to use the intel compilers, you need to contact cluster support. Might be useful to put together something detailing the pros/cons of using the different compilers.

File: wiki/macbookPro/OpenMP.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Use Homebrew to get g++ with OpenMP:
```
Comment (albeanth) [inline wiki/macbookPro/OpenMP.md:1]: something @harterj and I have been discussing are these mac package installer deals. There are a number of them. ID'ing which ones are better or worse than others at things would be useful to know. Personally, I don't have any of them. @harterj has a few installed on his machine.

**Comment by albeanth** [comment]
@dougwoods3 feel free to merge this in. The comments I made are more things we can tinker with down the road. I say just get it in so we can the general framework of the website side wiki going.

File: wiki/rogue/performance.md (hunk 1 above)
Comment (dougwoods3) [inline wiki/rogue/performance.md:1]: Good idea. I'll add the printinfo.cpp files and a link for now.

File: wiki/rogue/compilers.md (hunk 2 above)
Comment (dougwoods3) [inline wiki/rogue/compilers.md:4]: I will leave this one to future work.

File: wiki/macbookPro/OpenMP.md (hunk 3 above)
Comment (dougwoods3) [inline wiki/macbookPro/OpenMP.md:1]: I will leave this one to future work.

**Pull request merged**